### PR TITLE
Home screen banners

### LINKED ISSUE #74: Home screen banners
Add banners for no Bluetooth on/interacted with someone infected/reported

- Bluetooth Notif -> home page (not necessarily MVP)
- Stack Bluetooth and Infected banners if both are active

### PR DESCRIPTION
**Pull request by madim**
Shows "Turn on bluetooth" banner
Shows "Has possibly interacted with infected" banner
Handles "user tested positive" useflow
Ensures that contact event logging is enabled
Ensures that contact event data is fetched

### CODE REVIEW

File: app/src/main/java/org/covidwatch/android/di/AppModule.kt [hunk 1]
```diff
@@ -24,7 +25,18 @@ val appModule = module {
...
+
+    single {
+        CovidWatchDatabase.getInstance(androidContext())
+    }
+
+    single {
+        val database: CovidWatchDatabase = get()
+
+        database.contactEventDAO()
+    }
```
Comment (madim) [inline app/src/main/java/org/covidwatch/android/di/AppModule.kt:49]: you can now access CovidWatchDatabase and ContactEventDAO deps via DI

**Review by jamesjmtaylor — APPROVED** [review]
Could you link the issue that this is associated with in the PR comment for future reference?  Otherwise it looks good.  I was able to trigger the blue banner by launching with bluetooth off.
bluetooth

**Comment by nkumarcc** [comment]
Pulled this latest version, some things to note:
- "You may have been in contact" with I had to close then reopen the app to get (maybe just to prompt immediate Firebase pull)? Will see if Milen's PR fixes that.
- Couldn't get "You reported". Will try briefly debugging this before pulling

**Comment by nkumarcc** [comment]
Updated, turns out the way we're managing state is correct, we just needed the message to go into a banner and to change the home screen text.

**Pull request merged**